dashboard/test/models/script_test.rb

```diff
@@ -719,7 +719,7 @@ def populate_cache_and_disconnect_db
     }
 
     assert_equal 2, summary[:stages].count
-    assert_equal expected_peer_review_stage, summary[:peerReviewStage]
+    assert_equal expected_peer_review_lesson, summary[:peerReviewStage]
```

Comment:
Other spot where peerReviewStage shows up - see comment on peerReviewlesson

Reply:
Yea, this is the correct key, I'll fix the other one.